docs/packages.rst

```diff
@@ -117,42 +125,59 @@ community.
-3. Provide thorough test coverage and continuous integration of testing
-4. Maintain comprehensive documentation
-5. Establish a `code of
-   conduct <https://opensource.guide/code-of-conduct/>`__ for
-   contributors
-
-The `open-source guide <https://opensource.guide/>`__ provides some
-great advice on building and maintaining open-source projects.
+How to develop and maintain an open-source project is a large topic that extends beyond pangeo.
+The `pyOpenSci Package Development Guide`_ provides a comprehensive guide.
+
+Projects can submit their package for peer review from pyOpenSci.
 
 Best Practices for Pangeo Projects
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To address the needs of geoscience researchers, we have developed some
-additional recommendations:
-
-1. *Solve a general problem:* packages should solve a general problem
-   that is encountered by a relatively broad groups of researchers.
-2. *Clearly defined scope:* packages should have a clear and relatively
-   narrow scope, solving the specific problem[s] identified in the point
-   above (rather than attempting to cover every possible aspect of
-   geoscience research computing).
-3. *No duplication:* developers should try to leverage existing packages
-   as much as possible to avoid duplication of effort. (In early-stage
-   development and experimentation, however, some duplication will be
-   inevitable as developers try implementing different solutions to the
-   same general problems.)
-4. *Consume and Produce Xarray Objects:* Xarray data structures
-   facilitate mutual interoperability between packages.
-5. *Operate Lazily:* whenever possible, packages should avoid explicitly
-   triggering computation on Dask objects.
+In addition to general software best practices, there are some additional
+best-practices packages to work well with the pangeo ecosystem.
+
+1. *Solve a general problem*
+
+   Packages should solve a general problem that is encountered by a relatively
+   broad groups of users.
+
+2. *Clearly define a scope*
+ 
+   Packages should have a clear and relatively narrow scope, solving the
+   specific problem[s] identified in the point above (rather than attempting to
+   cover every possible aspect of geoscience research computing). By consuming
+   and producing standard data containers (see below) packages can compose
+   together to solve large problems.
+
+3. *Avoid duplication*
+
+   Developers should try to leverage existing packages as much as possible to
+   avoid duplication of effort. (In early-stage development and experimentation,
+   however, some duplication will be inevitable as developers try implementing
+   different solutions to the same general problems.)
+
+4. *Consume and produce standard data container*
+
+   Packages should consume and produce standard data containers like
+   ``xarray.Dataset`` and ``geopandas.GeoDataFrame``. This facilitate
+   interoperability between packages in the ecosystem.
+
+5. *Avoid data I/O where possible*
```

Comment:
@TomAugspurger in case you want to adjust the italicized text here? or i can adjust too

https://github.com/pyOpenSci/software-peer-review/pull/155/files#diff-4664f39bc8ad3c1af4ff7f0c43b027baef639404b6eca1ba861938ab4a1c3b88R39

Reply:
I am also very happy to adjust the language that I have now - please just say the word!